### PR TITLE
Feat: Landing page

### PR DESCRIPTION
**Pull request by pritam-kr**
- Created landing page 
- Created two-component
   - Navigation and hero
  
Screenshot (13)

### CODE REVIEW

File: src/Component/Hero/Hero.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    font-size: 3rem;
+}
+
+
+@media screen and (min-width: 676px) {
+    .hero-content {
+        font-size: 3rem;
+    }
+}
+
```
Comment (Shrinkhlarajpoot) [inline src/Component/Hero/Hero.css:25]: try to give media at the bottom of code.

File: src/Component/Hero/Hero.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import * as FaIcons from "react-icons/fa";
+import * as BiIcons from "react-icons/bi";
+
+const Hero = () => {
+    return(
+        <div className="hero-container">
+        <div className="hero-content">
+            <h2>Learn New trends with</h2>
+            <h1 className="site-name center"><FaIcons.FaPlayCircle /> PlayMe</h1>
+            <p className="text-lg" style={{marginTop: "1rem"}}>PlayMe is a learning platform for students.</p>
```
Comment (Shrinkhlarajpoot) [inline src/Component/Hero/Hero.jsx:12]: avoid inline styling if possible

**Review by Shrinkhlarajpoot — APPROVED** [review]

File: src/Component/Hero/Hero.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    margin: 3rem auto;
+}
+
+
+.searchbar-form {
+    border: 1px solid;
+    margin: 0 auto;
+    background-color: var(--white-text);
+    max-width: 45rem;
+
```
Comment (msdeshmukh009) [inline src/Component/Hero/Hero.css:36]: extra space here

**Review by msdeshmukh009 — APPROVED** [review]
The code is looking clean. Good to merge.

**Pull request merged**